runners/flink/src/main/java/org/apache/beam/runners/flink/translation/wrappers/streaming/DoFnOperator.java

```diff
@@ -643,6 +738,7 @@ public void onEventTime(InternalTimer<Object, TimerData> timer) throws Exception
 
   @Override
   public void onProcessingTime(InternalTimer<Object, TimerData> timer) throws Exception {
+    checkInvokeStartBundle();
```

Comment:
Maybe mention in `onEventTime()` that we don't have to cal `checkInvokeStartBundle()` because it's already called in `processWatermark*()`.